website/docs/r/terraform_version.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_terraform_version"
+sidebar_current: "docs-resource-tfe-terraform-version-x"
+description: |-
+  Manages Terraform versions
+---
+
+# tfe_terraform_version"
+
+Manages Terraform versions.
+
+## Example Usage
+
+Basic Usage:
+
+```hcl
+resource "tfe_terraform_version" "test" {
+  version = "99.99.99"
+  url = "https://tfe-host.com/path/to/terraform.zip"
+  sha = "e75ac73deb69a6b3aa667cb0b8b731aee79e2904"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `version` - (Required) A semantic version string in N.N.N or N.N.N-bundleName format.
+* `url` - (Required) The URL where a ZIP-compressed 64-bit Linux binary of this version can be downloaded.
+* `sha` - (Required) The SHA-256 checksum of the compressed Terraform binary.
+* `official` - (Optional) Whether or not this is an official release of Terraform. Defaults to "false".
+* `enabled` - (Optional) Whether or not this version of Terraform is enabled for use in Terraform Cloud. Defaults to "true".
```

Comment:
This is actually an issue in the API docs, too - this is true of TFE as well as TFC, even though the docs seem to indicate TFC only.

Reply:
So just update the docs to say `for use in Terraform Cloud or Enterprise` ?